Makefile

```diff
@@ -26,7 +26,8 @@ link_locations: import_directory flush_db
 .PHONY : update_db
 update_db : import_directory import_db auth_models.json flush_db link_locations import_google_docs
 	python manage.py loaddata auth_models.json
-        python manage.py make_flattened_views --recreate
+	python manage.py make_flattened_views --recreate
+	python manage.py update_countries_plus
```

Comment:
`update_countries_plus` imports geonames.org data and converts it into `Countries` objects that we use for search indexing. See: https://github.com/cordery/django-countries-plus#usage